main.py

```diff
@@ -1,54 +1,129 @@
 from pypanther import PantherLogType, PantherSeverity, get_panther_rules, register
+from pypanther.wrap import exclude, include
+from pypanther.get import table_print
+
 from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_account_discovery import AWSCloudTrailAccountDiscovery
 from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_stopped import AWSCloudTrailStopped
-from pypanther.rules.aws_cloudtrail_rules.aws_codebuild_made_public import AWSCloudTrailCodebuildProjectMadePublic
-from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_mfa import AWSConsoleLoginWithoutMFA
-from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_saml import AWSConsoleLoginWithoutSAML
-from pypanther.wrap import exclude, include
+from pypanther.rules.aws_cloudtrail_rules.aws_console_root_login import AWSConsoleRootLogin
+
+from rules.aws_rules.alb_high_400s import AWSALBHighVol400s
+from rules.custom_logs.ids_rules import HostIDSBaseRule, HostIDSMalware
+from helpers.cloud import prod_account_ids, account_lookup_by_id, update_account_id_tests
 
-from rules.aws_cloudtrail_rules.my_first_custom_rule import MyFirstCustomRule
-from rules.aws_cloudtrail_rules.my_first_tuned_rule import MyFirstTunedRule
 
 ########################################################
-## Variables
+## Importing Panther-managed Rules
+##
+## Use get_panther_rules() to return a list of Panther-managed rules.
+## Calling without arguments returns everything and passing attributes
+## like LogTypes, Severity, or Tags filters rules matching the values.
 
 onboarded_log_types = [
-    # replace with your log types
+    # Replace with your onboarded log types
     PantherLogType.AWS_CloudTrail,
+    PantherLogType.AWS_GuardDuty,
+    PantherLogType.Okta_SystemLog,
 ]
 
+# Get Panther built-in rules for onboarded log types
+base_rules = get_panther_rules(LogTypes=onboarded_log_types)
+
+## Get Panther-managed rules with certain severities
+# high_sev_rules = get_panther_rules(
+#     LogTypes=onboarded_log_types,
+#     Severity=[
+#         PantherSeverity.Critical,
+#         PantherSeverity.High,
+#     ],
+# )
+
+## Get all Panther-managed Rules
+# all_rules = get_panther_rules()
+
+
+########################################################
+## Filter/Dynamic Functions
+##
+## Add filters to include/exclude events from being evaluated by rules.
+
+
+def prod_account(event):
+    # TODO() Change this to use event.udm('account_id')
+    return event.get("recipientAccountId") in prod_account_ids
+
+
+sensitive_services = {"s3", "dynamodb", "iam", "secretsmanager", "ec2"}
+
+
+def guard_duty_sensitive_service(event):
+    service_name = event.deep_get("service", "action", "awsApiCallAction", "serviceName")
+    return any(service_name.startswith(service) for service in sensitive_services)
+
+
+def root_login_account(_, event):
```

Comment:
```suggestion
# this will be used to override a class method, so requires 2 args: self, event
def root_login_account(_, event):
```

Reply:
Nice suggestions - I'll add comments to those functions.